src/main/java/com/ibm/crypto/plus/provider/DHParameters.java

```diff
@@ -21,7 +21,8 @@
  * This class implements the parameter set used by the Diffie-Hellman key
  * agreement as defined in the PKCS #3 standard.
  */
-public final class DHParameters extends AlgorithmParametersSpi {
+public final class DHParameters extends AlgorithmParametersSpi implements java.io.Serializable {
+    private static final long serialVersionUID = 1L;
```

Comment:
Not positive this is safe to do? Does this class remain serializable? This comment also applies to a few other locations where the class no longer extends serializable.

Reply:
Not sure what you mean here. The class is serializable as long as all of its fields are serializable. Alternatively, we can add a `SuppressWarnings(serial)` or instruct the compiler to ignore such warnings.

Reply:
My question here is why did we make this class serializable? I assume we have a reason. If so then we should use a random serialVersionUID produced by something like `serialver` instead of using the value `1L`.

Reply:
I changed this class, because it is used by `DHPrivateKey` and `DHPublicKey`, which are serializable. The option there were:
- Make the parameters serializable too
- Specify the parameters as `transient` in the keys
- Ignore (i.e. suppress) the serialization warnings

I thought that was the best option, but we can always go for one of the other two.

Reply:
I updated the `serialVersionUID` using `serialver` as suggested.

Reply:
Updating the `serialVersionUID` looks good to me. This approach to adding Serializable to the class is as good as any of the other approaches you outlined above.